src/ddmd/constfold.d

```diff
@@ -1226,7 +1226,7 @@ extern (C++) UnionExp Index(Type type, Expression e1, Expression e2)
         uinteger_t i = e2.toInteger();
         if (i >= es1.len)
         {
-            e1.error("string index %llu is out of bounds [0 .. %llu]", i, cast(ulong)es1.len);
+            e1.error("string index %llu is out of bounds `[0 .. %llu]`", i, cast(ulong)es1.len);
```

Comment:
I think it would be better to highlight anything that comes from the user code, which in most cases means all %* arguments.